server/src/db/migrations/20240202160432_createReviews.cjs

```diff
@@ -0,0 +1,25 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
```

Comment:
this shouldn't be needed anymore? there is already a migration for the reviews table merged into the main branch

Reply:
removed